examples/Makefile

```diff
@@ -126,17 +126,7 @@ ifeq ($(PLATFORM),PLATFORM_DRM)
 endif
 
 # Define raylib release directory for compiled library
-RAYLIB_RELEASE_PATH 	?= $(RAYLIB_PATH)/src
-
-ifeq ($(PLATFORM),PLATFORM_WEB)
```

Comment:
Afaik, those lines are required to build examples with emscripten on Windows

Reply:
I think users should handle their PATH themselves, assuming the position of emsdk seems dangerous and bad practice.

Reply:
sorry but I prefer to keep those lines, they are useful for me and probably also to other users